pipeline/README.md

```diff
@@ -0,0 +1,33 @@
+# Data Ingestion Pipeline
+
+## Overview
+This is a data ingestion framework to intake enironment-specific data from the following FTP site: https://gml.noaa.gov/aftp/
+
+
+## Setup
+    # Actiate Virtual Environment
+    python3 -m venv venv
+
+    # Install Requirements
+    pip install requirements.txt
+    brew install adoptopenjdk11
+    brew install adoptopenjdk11
```

Comment:
In the future I'd like to see the setup split between OSX and Linux. we can attempt to build on my machine and write Linux setup steps.